### ISSUE TITLE
Better video about the DHT 

### ISSUE THREAD

**Issue by walkerlj0** [protocol/launchpad#106]
**Is your feature request related to a problem with the curriculum? Please describe.**
The current video at the bottom of this section (https://protocol-labs.gitbook.io/launchpad-curriculum/launchpad-learning-resources/ipfs/ipfs-public-dht) isn't very engaging, organized, and doesn't give very good context about the DHT

**Describe the solution you'd like**
Just a better presentation or video, or find a section of another video that gives a quicker, better explanation

**Where Can we find out more about this topic?**
To be researched. IPFS Docs

**Comment by mrodriguez3313** [comment]
Including resources for DHT: 
* https://docs.ipfs.io/concepts/dht/
* https://www.youtube.com/watch?v=KMmiAnMJU-c
* https://blog.ipfs.io/2020-07-20-dht-deep-dive/

Also I got a question @walkerlj0 about this post in the discord: https://discord.com/channels/806902334369824788/847893063841349652/991040117378527273

What's the status with the content of the md file? Like where did it move to?

**Comment by walkerlj0** [comment]
Hey @mrodriguez3313 can you link to the .md file? The discord link takes me to a channel, not a specific point in the discussion so I can't find it :/

**Comment by mrodriguez3313** [comment]
Yes sorry, this is the one https://hackmd.io/_WrVH75qSNy8XsjCklrP8A

Jorropo was saying that you were working on it. Im assuming to move the content around Launchpad curriculum.

**Comment by walkerlj0** [comment]
Hey,
I told Jorroporo that I would be happy to review the blog post, but I
didn't want to edit his work directly.

That content is a bit too long-form & needs a bit of organization to use in
Launchpad.

If you do see some value in pieces of it I would be happy to use it. I
would also be happy to edit his blog, but I don't have the ability to edit
it in HackMd

If you see some utility to it, we can try and find somewhere to put it in
the curriculum

**Comment by mrodriguez3313** [comment]
Yeah it wasn't complete and needed editing. But no worries, it didn't have any extra content that isn't already in the curriculum anyways. So I don't think we need it.
